clubs/views.py

```diff
@@ -1,9 +1,52 @@
+from django.contrib.auth import authenticate
+from django.contrib.auth.models import User
 from django.shortcuts import render
-from django.http import HttpResponse
+from django.urls import reverse
```

Comment:
Why it is here?
It is not used, please remove it and remember about using precommit before every pull request ;) 
It is worth to use github actions to run all linters on your code :) then you are sure that code was formatted